tests/suites/hup/suite.js

```diff
@@ -12,14 +12,35 @@ const { join } = require('path');
 const { homedir } = require('os');
 const Docker = require('dockerode');
 const exec = require('bluebird').promisify(require('child_process').exec);
+const Bluebird = require('bluebird');
 
 // Starts registry, uploads target image to registry
 const runRegistry = async (that, seedWithImage) => {
 	const docker = new Docker();
-	const registryImage = 'registry:2';
-
-	that.log('Pulling registry image');
-	await docker.pull(registryImage);
```

Comment:
Docker pull returns a stream, so even with an await the image won't be available on return of this function.